app/models/concerns/contact_record.rb

```diff
@@ -0,0 +1,7 @@
+module ContactRecord
+  extend ActiveSupport::Concern
```

Comment:
this line isn't strictly necessary, but adds functionality that could be useful in the future